README.md

```diff
@@ -255,27 +254,27 @@ streaming datasets.
 
 Installation
 ------------
-We recommend using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and conda to install Bokeh. Enter this command at a Bash or Windows command prompt:
+The easiest way to install Bokeh is using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system. This allows Bokeh and its required dependencies to be installed by entering the following command at a Bash or Windows command prompt:
```

Comment:
This seems more wordy and complicated to me. Again I prefer imperative call to action: *To do task X, perform action Y*.